src/models/developer.js

```diff
@@ -22,7 +22,7 @@ class Developer {
         if(index > -1){
             this._currentProjects.splice(index, 1);
         }
-        return 'Project does not exist';
+        return false;
```

Comment:
Jeg har lige bemærket at de her metoder returnere `false` uanset hvad :P Du skal nok have en return true inde i din is-statement hvor elementet bliver slettet.